apps/backend/prisma/schema.prisma

```diff
@@ -37,17 +37,27 @@ model User {
   isSchResident      Boolean
   isActiveVikStudent Boolean
   roomNumber         Int?
-  profileImage       Bytes?
   canHelpNoobs       Boolean             @default(false)
   publicDesc         String?
   createdAt          DateTime            @default(now())
   updatedAt          DateTime            @updatedAt
   vikStatusUpdatedAt DateTime            @default(now())
+  profilePicture     ProfilePicture?
   applicationPeriods ApplicationPeriod[]
   applications       Application[]
   posts              Post[]
 }
 
+model ProfilePicture {
```

Comment:
I put it into a separate table, because it would be read from the database each time a user is fetched (because prisma uses select * by default)